src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -763,14 +823,14 @@ void BLEManagerImpl::HandleBootEvent(void)
     PlatformMgr().ScheduleWork(DriveBLEState, 0);
 }
 
-void BLEManagerImpl::HandleConnectEvent(SilabsBleWrapper::sl_wfx_msg_t * evt)
+void BLEManagerImpl::HandleConnectEvent(const SilabsBleWrapper::sl_wfx_msg_t & evt)
 {
-    AddConnection(evt->connectionHandle, evt->bondingHandle);
+    AddConnection(evt.connectionHandle, evt.bondingHandle);
     PlatformMgr().ScheduleWork(DriveBLEState, 0);
 }
 
 // TODO:: Implementation need to be done.
```

Comment:
I think this comment can be removed now.

Reply:
Remvoed